tests/run

```diff
@@ -71,7 +71,7 @@ function run_test()
   )
 }
 
-install_typst
+# install_typst
```

Comment:
Is this problematic on Windows?

Reply:
:/ Kind of, it got annoying that it had to install typst every time. Theres a few other changes I have to make to it to get it to work. Too be honest we need to overhaul the whole testing system.